README.md

```diff
@@ -264,13 +245,32 @@ The most common CSVKit commands that we use in data processing are [```in2csv```
 	#	csvjoin -c [column in csv1],[column in csv2] [csv1 filepath] [csv2 filepath] > [target filepath]
 	
 	# REAL EXAMPLE	
-	%hourly.joined.csv: %hourly.csv
-		csvjoin -c 1,2 build/$(notdir $?) raw/stations.csv > build/$(notdir $@)
+	%hourly.joined.csv: %hourly.csv raw/stations.csv
+		csvjoin -c 1,2 build/$(notdir $?) $(word 2,$^) > build/$(notdir $@)
 	```
 	
 	- ```%``` is a pattern that matches any nonempty substring (making this an implicit rule)
 	- ```$?``` refers to a dependency that's newer than its corresponding target
 	
+### ETL Workflow Directory Structure
+
+In the case that a project has multiple separate data components, you can define a master makefile at the root of the repository, along with sub-directories that each have a sub-makefile at the root. When using this type of nested structure, all data processing/transformation should be defined in the sub-makefiles - the master makefile should only handle setting up the environment, defining variables/targets used by multiple sub-makefiles, & calling sub makefiles.
+
+```
+|-- Makefile  # the master makefile
+|-- README.md
+|-- data
+|   |-- <sub-directory for a data processing component>
```

Comment:
Seems to me that the source files and the clean files should all be in the same subdir. Intermediate files should also live in same dir, but should be automatically cleaned up or cleaned up with 'make clean'